.gitlab-ci.yml

```diff
@@ -9,6 +9,7 @@ noether-rocm:
   script:
 # Compilers
     - export COVERAGE=1 CC=gcc CXX=g++ FC=gfortran HIPCC=hipcc
+    - echo "---------- nproc-----------" && NPROC_CPU=$(nproc) && NPROC_GPU=8
```

Comment:
Should `NPROC_GPU` be a minimum of `$(nproc)` and some static (or determined) maximum?

Reply:
agreed and updated